xbmc/guilib/MatrixGLES.cpp

```diff
@@ -326,10 +326,7 @@ void CMatrixGL::PrintMatrix(void)
 
 void CMatrixGLStack::Load()
 {
-#ifdef HAS_GL
-  glMatrixMode(m_type);
```

Comment:
`m_type` is now unused at least on osx

Reply:
This entire file is a hack: MatrixGLES and used by OpenGL. Load() is empty now and can be removed but this would reslts in more changes in GLES only code. I think it is best to do this in a separate PR.